pkg/migrations/op_set_unique.go

```diff
@@ -150,6 +150,9 @@ func (o *OpSetUnique) Validate(ctx context.Context, s *schema.Schema) error {
 	if o.Name == "" {
 		return FieldRequiredError{Name: "name"}
 	}
+	if err := validateName(o.Column); err != nil {
+		return err
+	}
```

Comment:
I don't think it makes sense to check the length of the column name on the operation here as it must already exist in the database. The check can be removed.